project-base/storefront/helpers/cookies/cookiesStoreUtils.ts

```diff
@@ -0,0 +1,34 @@
+import { getCookies, setCookie } from 'cookies-next';
+import { OptionsType } from 'cookies-next/lib/types';
+import { GetServerSidePropsContext } from 'next';
+import { useEffect } from 'react';
+import { CookiesStoreState, useCookiesStore } from 'store/useCookiesStore';
+
+export const COOKIES_STORE_NAME = 'cookiesStore' as const;
```

Comment:
No need to export it, it's only used here

Reply:
Yup, removed.